### PR TITLE
Add global listener for Call Visualizer engagement start event

### PR DESCRIPTION
**Pull request by gugalo**
**Jira issue:**
https://glia.atlassian.net/browse/MOB-1934

### CODE REVIEW

File: widgetssdk/src/main/java/com/glia/widgets/core/callvisualizer/domain/CallVisualizer.java [hunk 1]
```diff
@@ -39,4 +39,14 @@ public interface CallVisualizer {
      */
     void addVisitorContext(String visitorCodeContext);
 
+    /**
+     * Sets callback that will called when Call Visualizer engagement is started.
```
Comment (andrews-moc) [inline widgetssdk/src/main/java/com/glia/widgets/core/callvisualizer/domain/CallVisualizer.java:43]: that will called => that will be called

**Review by andrews-moc — APPROVED** [review]

File: widgetssdk/src/main/java/com/glia/widgets/core/callvisualizer/domain/CallVisualizer.java (hunk 1 above)
Comment (gugalo) [reply widgetssdk/src/main/java/com/glia/widgets/core/callvisualizer/domain/CallVisualizer.java]: Fixed: a66308e

**Comment by gugalo** [comment]
I will close this PR because of the wrong ticket in the branch name. Will also change it in PR description and commit description

**Comment by gugalo** [comment]
Created new one with fixes: https://github.com/salemove/android-sdk-widgets/pull/490

**Pull request closed without merging**